Test behavior of media capture from a cross-origin video

dontcallmedom opened the pull request:
<!-- Reviewable:start -->

<!-- Reviewable:end -->

dontcallmedom committed:
Test behavior of media capture from a cross-origin video

w3c-bots commented:
# Build PENDING

Started: None
Finished: None





View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/19604)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/289035212)

yellowdoge commented on mediacapture-fromelement/cross-origin.html:
```diff
@@ -0,0 +1,41 @@
+// Run captureStream() on cross-origin <video> content
+  async_test(function() {
+    const video = document.createElement('video');
+    video.src = location.origin.replace("//", "//www1.") + "/media/test-v-128k-320x240-24fps-8kfr.webm";
```
test-v-128k-320x240-24fps-8kfr.webm is about 38KB in size. If we don't care about the contents, we could consider using white.webm which is just 10KB, and will perhaps avoid flakes in slow bots.

zcorpan commented:
@dontcallmedom can you rebase?

alvestrand commented:
Closing and reopening to refresh status.

dontcallmedom committed:
Use smaller video source

As requested in https://github.com/web-platform-tests/wpt/pull/7862/files#r145160486
